src/test/resources/org/checkstyle/autofix/recipe/hexliterialcase/hexliteralcase/InputHexLiteralCase.java

```diff
@@ -0,0 +1,23 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="com.puppycrawl.tools.checkstyle.checks.HexLiteralCaseCheck"/>
+  </module>
+</module>
+
+*/
+
+package org.checkstyle.autofix.recipe.hexliterialcase.hexliteralcase;
+
+public class InputHexLiteralCase {
```

Comment:
could you add few more use cases like:
1. `int val = 0x1b, sum = 0xff;`
2. `functionCall(0x7fff, 0xab)`